source/backend/api/Areas/DocumentGeneration/DocumentGenerationController.cs

```diff
@@ -1,16 +1,19 @@
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Pims.Api.Models;
+using Pims.Api.Policies;
 using Pims.Api.Services;
+using Pims.Dal.Security;
 using Swashbuckle.AspNetCore.Annotations;
 
 namespace Pims.Api.Controllers
 {
     /// <summary>
     /// DocumentGenerationController class, provides endpoints to handle document generation requests.
     /// </summary>
-    //[Authorize]
+    [Authorize]
```

Comment:
I see you enabled authorization here but still relevant claims on specific methods below are commented out. Shouldn't they be enabled as well?

Reply:
I think Devin has a PR for those claims types. Until that is merged the methods below cannot be commented out.

Reply:
Fair enough, but still confused as to why your PR cannot uncomment the methods below but your new code can do:


```csharp
 this.User.ThrowIfNotAuthorized(Permissions.DocumentView);
```

Aren't they the same document permissions?

Reply:
Like here in Line 44 `//[HasPermission(Permissions.DocumentView)]`

Reply:
My PR shouldn't affect any of the claims in this sprint. It should allow a developer in the next sprint to allow document generation to normal (non-admin) users.

Reply:
Probably both need to be changed. The new permission is DocumentGenerate.